docs/installation/opni/backends.md

```diff
@@ -14,7 +14,30 @@ You can currently create the following backends:
 TODO Joe
 </TabItem>
 <TabItem value="opni-logging" label="Opni Logging">
+To enable the Opni Logging backend, select `Logging` under `Backends` in the left hand navigation menu of the Opni Admin UI, and hit the Enable button.  You will be presented with a number of details to fill out.
+<br/>
+<br/>
+
+#### External URL
+This is the URL that the Opensearch API will be exposed on.  You will need to manually expose this URL using either an Ingress or Load Balancer service.
```

Comment:
we can add some examples here or in the UI though? Maybe in the UI, Grafana has this feature when I tried to add new data source.
From my user experience, suppose my hostname is `myopni.xyz` and I exposed the dashboard at port 5601 --> `myopni.xyz:5601` won't work and it took me a while to figure out it should be `https://myopni.xyz:5601`.

Reply:
This external URL is for the opensearch API, not the Dashboards UI.  I'm hesitant to add examples as there's quite a few permutations of how this could be exposed and I think we risk confusing the issue.

Rather I'd prefer to wait until we have a how to section and provide examples of setting up ingresses there.

Reply:
same applies to the API url --> it has to be `https://myopni.xyz:9200` instead of `myopni.xyz:9200`. But yeah a `how to` section would be good